pkg/codegen/pcl/binder_resource.go

```diff
@@ -59,19 +59,29 @@ func (b *binder) bindResourceTypes(node *Resource) hcl.Diagnostics {
 		pkg, isProvider = name, true
 	}
 
-	pkgSchema, ok := b.options.packageCache.entries[pkg]
-	if !ok {
-		return hcl.Diagnostics{unknownPackage(pkg, tokenRange)}
+	var pkgSchema *packageSchema
+	if pkg != "pulumi" && module != "pulumi" {
```

Comment:
Could we switch the order of this conditional so it matches the conditional on line 76.